frontend/src/components/HeaderLogo.vue

```diff
@@ -6,18 +6,16 @@ export default {
 
 <template>
   <header>
-    <img src="../assets/ee_logo.png" />
+    <img src="../assets/img/rasporedar_logo.svg" />
   </header>
 </template>
 
 <style scoped>
 img {
-  width: 200px;
+  width: 75px;
 }
-@media (max-width: 700px) {
-  img {
-    display: block;
-    margin: auto;
-  }
+
+header {
+  margin: 16px 0;
```

Comment:
Hint: it is best not to target tag names directly when scoped is used. [Here](https://vuejs.org/api/sfc-css-features.html#scoped-css) you can find more information about why.